### PR TITLE
lint error fix

### PR DESCRIPTION
**Pull request by ateeqrehmani**

### CODE REVIEW

File: etc/tronUtils.js [hunk 1]
```diff
@@ -235,6 +235,32 @@ async function getAllergyForm(address) {
...
+        let tronWeb = new TronWeb(fullNode, solidityNode, eventServer);
+        let account = await tronWeb.trx.getAccount(address);
+        let balance = 0;
+        if (account == "") {
+            console.log('account info blank');
+            return getTRC10TokenBalance(privateKey, address);
+        }
+        else if (account.assetV2) {
+            for (let i = 0; i < account.assetV2.length; i++) {
+                if (account.assetV2[i].key == '1001581') {
```
Comment (atif-rehman-ideofuzion) [inline etc/tronUtils.js]: Add comment for static comparisons.

Comment (ateeqrehmani) [reply etc/tronUtils.js]: added

File: sendNewEHRtoVoters.js [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+		//Getting voter data
+		options.uri = `${apiUrlForVotersList}?limit=50&candidate=${process.env.VOTER_ACCOUNT}`
+		const response = await rp(options)
+
+		for (let i = 0; i < response.data.length; i++) {
+			const oldTokenBalance = await tronUtils.getHealthportOldTokenBalance(response.data[i].voterAddress)
+			const newTokenBalance = await tronUtils.getTRC10TokenBalance('', response.data[i].voterAddress)
+			if (newTokenBalance == 0 && oldTokenBalance > 0 && (newTokenBalance != oldTokenBalance)) {
+				//Excluding potential hacker
+				if (response.data[i].voterAddress != 'TB5ZDGoqiL89HiNMWo26b6ciZUUv6QsfWQ') {
```
Comment (atif-rehman-ideofuzion) [inline sendNewEHRtoVoters.js]: Ignore case sensitivity for this address.

Comment (ateeqrehmani) [reply sendNewEHRtoVoters.js]: toLowerCase added for comparison

**Pull request merged**